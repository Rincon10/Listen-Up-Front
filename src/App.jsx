import AppRouter from 'components/routers/AppRouter';
import './css/index.css';

function App() {
    return (
        <div className="App">
            <AppRouter />
        </div>
    );
}

export default App;
