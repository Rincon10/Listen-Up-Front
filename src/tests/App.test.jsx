import React from 'react';
import { shallow } from 'enzyme';
import { describe, expect, test } from '@jest/globals';
import '@testing-library/jest-dom';
import App from '../App';

describe('Test to the component <App />', () => {
    test('The component <App /> should render propertly', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});
