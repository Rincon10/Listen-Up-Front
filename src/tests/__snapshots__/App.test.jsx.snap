// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to the component <App /> The component <App /> should render propertly 1`] = `
<main
  id="main-container"
>
  <ContextProvider
    value={
      Object {
        "dispatch": [Function],
        "user": Object {
          "logged": false,
        },
      }
    }
  >
    <AppRouter />
  </ContextProvider>
</main>
`;
